themes-default/slim/src/index.js

```diff
@@ -114,7 +121,17 @@ $.fn.extend({
     }
 });
 
-if (!document.location.pathname.includes('/login')) {
+// @FIXME: Workaround just for the time being!
+const log = {
```

Comment:
What's this needed for?

Reply:
`log` is defined, I chose to make a workaround instead of changing the calls to `console.[..]`, like you did in #4869.

Reply:
I actually think the `log` should have been removed a while ago. I removed the old logger months and months ago.